utilities/src/main/java/io/syndesis/qe/bdd/validation/MetricsValidationSteps.java

```diff
@@ -0,0 +1,64 @@
+package io.syndesis.qe.bdd.validation;
+
+import org.assertj.core.api.Assertions;
+
+import java.security.GeneralSecurityException;
+import java.util.Date;
+import java.util.List;
+
+import cucumber.api.java.en.Then;
+import io.syndesis.model.integration.Integration;
+import io.syndesis.model.metrics.IntegrationMetricsSummary;
+import io.syndesis.qe.endpoints.IntegrationsEndpoint;
+import io.syndesis.qe.endpoints.MetricsEndpoint;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class MetricsValidationSteps {
+
+	private IntegrationsEndpoint integrationsEndpoint;
+	private MetricsEndpoint metricsEndpoint;
+
+	@Then("^validate that number of all messages through integration \"([^\"]*)\" is greater than \"([^\"]*)\", period in ms: \"([^\"]*)\"$")
+	public void validateThatNumberOfAllMessagesOfIntegrationIsGreaterThanPeriodInMs(String integrationName, Integer nr, Integer ms) throws InterruptedException {
+		Thread.sleep(ms + 1000);
+//		0.  get integration id.
+		String id = this.getIdByIntegrationName(integrationName);
+		Assertions.assertThat(id).isNotNull();
+
+//		1. get metrics:
+		try {
+			metricsEndpoint = new MetricsEndpoint();
+		} catch (GeneralSecurityException e) {
```

Comment:
Pls initialize both endpoints in contructor. @tplevko and maybe I'd like to swallow that exception in AbstractEndpoint :)

Reply:
ok